jcasc/jenkins.yaml

```diff
@@ -74,6 +78,15 @@ security:
 unclassified:
   location:
     url: "${HTTP_PROTO}://${HOSTNAME}:${HOSTPORT}/"  # Env var
+  throttleJobProperty:
+    # https://plugins.jenkins.io/throttle-concurrents/#plugin-content-global-configuration
+    categories:
+    - categoryName: "daccs-e2e"
+      maxConcurrentTotal: 0
+      maxConcurrentPerNode: ${MAX_CONCURRENT_JOBS}
+      nodeLabeledPairs:
+      - throttledNodeLabel: "docker"
+        maxConcurrentPerNodeLabeled: 1
```

Comment:
What the difference between ` maxConcurrentPerNode` and `maxConcurrentPerNodeLabeled`?

You are not only throttling per node, also throttling per label?  And the label value is hardcode?